indico/htdocs/sass/modules/_papers.scss

```diff
@@ -41,3 +43,81 @@
         }
     }
 }
+
+.display-contribution-list {
+    @extend %font-family-modern-body;
+    .contribution-row {
+        padding-left: 10px;
+        border-left: 3px solid $pastel-gray;
+        margin-bottom: 1em;
+        &:hover {
+            border-color: $blue;
+        }
+        .row-title {
+            font-size: 14pt;
+            @include border-bottom();
+        }
+        .submitter {
+            color : $light-black;
+            margin-top: 7px;
+        }
+        .label {
+            border: 0;
+            margin-top: 7px;
+            display: inline-block;
+            padding: 5px 7px;
+
+            &.review-role {
+                margin-right: 10px;
+                background-color: white;
+                color: $light-black;
+                border: 1px solid $gray;
```

Comment:
I think this can be done by `@include border-all($gray)` (or `@include border-all()` if `$pastel-gray` suits you).